diagnostic.md

```diff
@@ -7,40 +7,40 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git clone git@github.com:joechristman/git-basics-diagnostic.git
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+cd into cloned repo and created a 'response' branch
 
 3.What command do we use to create a new branch? Name this branch `response`
     in your answer. Then, how do we switch to that branch?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git branch response
```

Comment:
That is to create the branch. But we will have to switch to it by `git checkout response`.

A short cut if you haven't seen it is `git checkout -b response`. This will create the branch and check it out.